charts/external-dns/values.yaml

```diff
@@ -296,7 +296,7 @@ provider:  # @schema type: [object, string];
       relabelings: []
 
 # -- Extra arguments to provide to _ExternalDNS_.
-extraArgs: []  # @schema type: [array, null]; item: string; uniqueItems: true;
+extraArgs: {}  # @schema type: [array, null, object]; item: string; uniqueItems: true;
```

Comment:
I think we need to improve the docs comment to cover the potential for different types.

Reply:
I pushed a change, please check if that's how you wanted this.